test/hotspot/jtreg/runtime/appcds/MoveJDKTest.java

```diff
@@ -102,9 +105,20 @@ public static void main(String[] args) throws Exception {
                                             "-Xlog:class+path=info",
                                             "-version");
             OutputAnalyzer out = TestCommon.executeAndLog(pb, "exec-dst");
+            out.shouldHaveExitValue(0);
             out.shouldNotContain("shared class paths mismatch");
```

Comment:
Includes the changes from [JDK-8272335](https://bugs.openjdk.org/browse/JDK-8272335) ( commit [75a0642](https://github.com/openjdk/jdk/commit/75a06421e596b0e0006120a13836ec65a3ab9d46) ) on this file